src/components/hello/Hello.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const time = new Date().getHours();
+let text = "Good night";
+if (time > 3) {
+    if (time < 12) {
+        text = "Good morning";
+    } else {
+        if (time < 18) {
+            text = "Good afternoon";
+        } else {
+            if (time < 22) {
+                text = "Good evening";
+            }
+        }
+    }
+}
```

Comment:
It is difficult to understand these nested conditions. Please make it simplier, no need _if_ inside other _if_ (try to use logic operators `&&` and `||`)